docs/data-sources/google_cost_credentials.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
are those here by mistake (the data docs files)

Reply:
didn't touch those files, maybe generated on the repo and then pull to my local

Reply:
they aren't in `main` thats why we see them as changes